es-bs-base/scss/utilities/_sizing.scss

```diff
@@ -18,3 +18,22 @@
 
 .vw-100 { width: 100vw !important; }
 .vh-100 { height: 100vh !important; }
+
+// Width Auto Utils
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+    .w#{$infix}-auto { width: auto !important; }
+  }
+}
+
+// Min-width EM utils
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+    // Add min-width (in rem) utilities, especially useful for buttons
+    @for $i from 4 through 7 {
+      .min-width#{$infix}-#{$i} { width: #{$i}em !important; }
```

Comment:
Per the [recipe in es-form](https://github.com/EnergySage/es-ds/blob/main/es-design-system/pages/organisms/es-form.vue#L83-L101) I found that a 2em spinner can extend beyond a "width auto" button. A few min-width "in-em" utilities I find really helpful for larger viewports, especially in conjunction with the `w-auto` utility classes.